google/cloud/forseti/services/base/config.py

```diff
@@ -71,79 +71,42 @@ class AbstractInventoryConfig(dict):
 
     @abc.abstractmethod
     def get_root_resource_id(self):
-        """Returns the root resource id.
-
-        Raises:
-            NotImplementedError: Abstract.
-        """
-        raise NotImplementedError()
+        """Returns the root resource id."""
```

Comment:
Do we need a pass at the end of the methods? 

e.g.

 def get_root_resource_id(self):
        """Returns the root resource id."""
        pass